README.md

```diff
@@ -24,16 +24,20 @@ npm ci
 ```
 **NOTE** `ci` (as opposed to `install`) is a 'clean install' which ensures versions exactly match package-lock.json).
 
-#### Boot the Server (sails backend)
+#### Boot the App
```

Comment:
While we're at it, can we say 'Start' instead of 'Boot' in these headers? I think aligning the language with the command names could help make them easier to remember